yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
please remove

Reply:
fixed in [6dc9ee2](https://github.com/rhdevs/rhapp/pull/628/commits/6dc9ee2c7232a5716e26229b75394d57a377edcc)